runners/flink/src/main/java/org/apache/beam/runners/flink/metrics/FlinkMetricContainer.java

```diff
@@ -77,12 +82,72 @@ public FlinkMetricContainer(RuntimeContext runtimeContext) {
     this.metricsAccumulator = (MetricsAccumulator) metricsAccumulator;
   }
 
-  MetricsContainer getMetricsContainer(String stepName) {
+  public MetricsContainer getMetricsContainer(String stepName) {
     return metricsAccumulator != null
         ? metricsAccumulator.getLocalValue().getContainer(stepName)
         : null;
   }
 
+  /**
+   * Parse a {@link MetricName} from a {@link
+   * org.apache.beam.model.fnexecution.v1.BeamFnApi.MonitoringInfoUrns.Enum}
+   *
+   * <p>Should be consistent with {@code parse_namespace_and_name} in monitoring_infos.py
```

Comment:
Wanted to flag this for @ajamato in particular as the author of [the python code I mimicked here](https://github.com/apache/beam/blob/6b7cf422733a82aabb8e257e975d9e9c5e785376/sdks/python/apache_beam/metrics/monitoring_infos.py#L228-L236).

How important is it that the python and java SDKs parse metric URNs into a {namespace, name} pairs in the same way?

From what I can tell, It may not be critical, though we should still probably agree on one semantic interpretation of what is "namespace" vs "name" of e.g. [`"beam:metric:element_count:v1"`](https://github.com/apache/beam/blob/master/model/fn-execution/src/main/proto/beam_fn_api.proto#L360-L361).

With `monitoring_infos.py` as the reference implementation, the answer is {`"beam"`, `"metric:element_count:v1"`}; shall I encode that in a comment in `beam_fn_api.proto`, and reference it here?